README.md

```diff
@@ -1,15 +1,10 @@
 # Github Pull Request Counter
 
-A chrome extension to remove mental load when working with pull requests.
+A browser extension to remove mental load when working with pull requests.
 It has been published in
 the [Chrome Web Store](https://chrome.google.com/webstore/detail/github-pull-request-count/eeejbcmnmgogpkgeinlbchoafjjbegmi)
-to be installed with one click.
-
-## Firefox port
-
-There is work in progress for a Firefox port of the extension in the
-[main-firefox](https://github.com/renuo/github-pull-request-counter/tree/main-firefox)
-branch.
+and the [Firefox Add-ons Store](https://addons.mozilla.org/en-US/firefox/addon/github-pull-request-counter/) to be
```

Comment:
Doesn't exist yet I guess?

Reply:
The extension is still "in review" so you'll only see the product page when you're logged in with the renuo account. Will be live soon :)